rulebook.tex

```diff
@@ -368,10 +368,17 @@ \subsection{The Task}
 \subsection{Agreements \& Regulations}
 \label{sec:agreements}
 \paragraph{Standardized Robotic Platform}
-In the \ac{RCLL}, all teams are obliged to use the Robotino robotic system
-from Festo Didactic SE with certain freedoms and limitations. This
+In the \ac{RCLL}, the Robotino robotic system
+from Festo Didactic SE is used as common base platform that can be used with
```

Comment:
Suggestion: 
In the \ac{RCLL}, the Robotino robotic system from Festo Didactic SE is used as the common base platform. It can be used with certain freedoms and limitations regarding hardware and software modifications and additions, which will be outlined in detail later.

Reply:
Yea makes sense, i fixed this in https://github.com/robocup-logistics/rcll-rulebook/pull/46/commits/eeb4b8fb8c5de3c468d9d464c0e31df1c13d8282